SimCalorimetry/HcalZeroSuppressionProducers/python/hcalDigisRealistic_cfi.py

```diff
@@ -24,8 +24,8 @@
 from Configuration.Eras.Modifier_phase2_hcal_cff import phase2_hcal
 phase2_hcal.toModify( simHcalDigis,
                              useConfigZSvalues = cms.int32(1),
-                             HBlevel = cms.int32(16),
-                             HElevel = cms.int32(16),
-                             HOlevel = cms.int32(16),
-                             HFlevel = cms.int32(16)
+                             HBlevel = cms.int32(8),
```

Comment:
I mean something a bit different. As written, the code is equivalent to

simHcalDigis.HBlevel=cms.int32(8)
if phase2: simHcalDigis.HBlevel=cmsint32(8)

so naively line 27,29,30 can just be dropped. (?)